src/include/optimizer/operators.h

```diff
@@ -147,13 +147,13 @@ class LogicalGroupBy : public OperatorNode<LogicalGroupBy> {
 class LogicalInsert : public OperatorNode<LogicalInsert> {
  public:
   static Operator make(
-      storage::DataTable *target_table, std::vector<char *> *columns,
-      std::vector<std::vector<peloton::expression::AbstractExpression *> *> *
+      storage::DataTable *target_table, std::vector<std::unique_ptr<char[]>> *columns,
+      std::vector<std::unique_ptr<std::vector<std::unique_ptr<peloton::expression::AbstractExpression>>>> *
```

Comment:
You are already in the `peloton` namespace, so this seems unnecessary. Change to `expression::AbstractExpression`